components/supervisor-api/go/control.pb.gw.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Copyright (c) 2024 Gitpod GmbH. All rights reserved.
```

Comment:
@filiptronicek I'd love to approve: Could you remove all of these changes from the PR? :pray: